static/editor.js

```diff
@@ -1,20 +1,24 @@
 (function () {
-	let changed = false;
+	window.hyphaChanged = false;
 	let textarea = document.querySelector('.edit-form__textarea');
 	let form = document.querySelector('.edit-form');
 
+	// some browsers tend to cache user's text, so we'd better keep
+	// the original one for later use (e.g. draft notifications)
+	window.hyphaOriginal = textarea.value;
```

Comment:
Why do we need this variable?

Reply:
A little feature for draft support. I can cut it and present in another PR if it's desired.

Reply:
Yeah, would've been better to not add more code until it's needed and keep the PRs atomic.

Reply:
Force-pushed the fix